src/homelypy/devices.py

```diff
@@ -132,6 +132,14 @@ class MotionSensorMini(Device):
     alarm: MotionSensorState
 
 
+@dataclass
+class MotionSensor2(Device):
```

Comment:
There is no need to duplicate the MotionSensorMini device when the features of the device are identical. It sufficient to add the new device model name to the device map and point to the existing MotionSensorMini device.